src/parse.ts

```diff
@@ -55,6 +55,39 @@ export function parseMessage(
   return output as ParsedMessage;
 }
 
+export function parseTimestamp(
+  definition: MessageDefinition,
+  definitions: Map<string, MessageDefinition>,
+  view: DataView,
+  offset = 0,
+): bigint {
+  let timestamp: bigint | undefined;
+
+  let curOffset = offset;
+  for (const field of definition.fields) {
+    if (field.name.startsWith("_")) {
+      continue;
+    }
```

Comment:
what is special about fields that start with an underscore? The if condition here could be safely remove because the name will never match "timestamp"

Reply:
No idea, it was already present in the previous parse code and didn't have a comment.